src/app/hooks/useTransactions.ts

```diff
@@ -17,16 +18,38 @@ export const useTransactions = () => {
           accountId,
           limit,
         });
+        const fetchInvoice = await api.getInvoices({ isSettled: true, limit });
```

Comment:
Talking with @bumi , he suggested we add a new connector method to get all transactions. Then the alby connector can be simplified because the logic can be implemented in the backend.

Should we do this as a follow up PR?

Reply:
we also need to discuss feasibility for this for other connectors as well. something that could be done as follow up pr. yup!